src/pointarray.cpp

```diff
@@ -44,8 +44,8 @@ struct OctreeNode
     size_t endIndex;         ///< End index of points in this node
     mutable size_t nextBeginIndex; ///< Next index for incremental rendering
     Imath::Box3f bbox;       ///< Actual bounding box of points in node
-    V3f center;              ///< Centre of the node
-    float radius;            ///< Distance to centre to node edge along an axis
+    V3f center;              ///< center of the node
```

Comment:
Yes, it's slightly depressing, but crushed by the weight of 1000 APIs the only real option seems to be to give in :-)